lib/internal/Magento/Framework/File/Test/Unit/Transfer/Adapter/HttpTest.php

```diff
@@ -34,7 +43,11 @@ protected function setUp()
             ['setHeader', 'sendHeaders', 'setHeaders']
         );
         $this->mime = $this->createMock(\Magento\Framework\File\Mime::class);
-        $this->object = new Http($this->response, $this->mime);
+        $this->request = $this->createPartialMock(
+            \Magento\Framework\App\Request\Http::class,
```

Comment:
Please, avoid FQCN usage.

Reply:
Could you elaborate, please? I'm not sure what you mean by this.

Reply:
`use Magento\Framework\App\Request\Http` in the import section and just `Http` class name where you need it.